tests/test_management.py

```diff
@@ -63,6 +63,14 @@
 PAST_TIMESTAMP_FORMATTED = PAST.strftime('%F')
 FUTURE = NOW + DAY_DELTA
 
+# Silence noisy logs
+LOG_OVERRIDES = [
+    ('stevedore.extension', logging.ERROR),
+]
+
+for log_name, log_level in LOG_OVERRIDES:
+    logging.getLogger(log_name).setLevel(log_level)
```

Comment:
**Note to Reviewers:** getting `stevedore.extension: DEBUG: found extension EntryPoint.parse('block-v1 = opaque_keys.edx.locator:BlockUsageLocator')` which causing the test failures so disabled the debug logs. Above code is proudly copied from https://github.com/edx/edx-platform/pull/5459